src/main/services/user_account/ResetApiCredsService.kt

```diff
@@ -1,2 +1,47 @@
 package main.services.user_account
 
+import framework.models.idValue
+import kotlinserverless.framework.services.SOAResult
+import kotlinserverless.framework.services.SOAResultType
+import main.daos.*
+import main.services.transaction.GenerateTransactionService
+import java.lang.Exception
+
+/**
+ * This service will be used to reset a user account's ApiCreds
+ */
+object ResetApiCredsService {
+     fun execute(userId: Int) : SOAResult<ApiCredNamespace> {
```

Comment:
change this to userAccount rather than userId since we should have the userAccount already